File: backend/tests/test_failing.py
def test_failing_addition():
    """This test should FAIL - wrong expected result"""
    assert 1 + 1 == 3  # This is wrong!

def test_failing_string():
    """This test should FAIL - string mismatch"""
    name = "Taboor"
    assert name == "WrongName"  # This will fail

File: frontend/test_failing.js
// Simple failing JavaScript test
function testFailingAddition() {
    if (2 + 2 !== 5) {  // This is wrong!
        throw new Error("Failing test: 2+2 should equal 5 (this is wrong on purpose)");
    }
}

// This will throw an error and fail the test
testFailingAddition();